lib/data_migrate/config.rb

```diff
@@ -12,11 +12,12 @@ def config
   end
 
   class Config
-    attr_accessor :data_migrations_path, :db_configuration
+    attr_accessor :data_migrations_path, :db_configuration, :db_name
 
     def initialize
       @data_migrations_path = "db/data/"
       @db_configuration = nil
+      @db_name = nil
```

Comment:
```suggestion
      @spec_names = nil
```
where nil means we want to run all spec_names. expect spec_names to be an array of strings.

Reply:
sorry, i'm a bit confused by this, what is `spec` in this case? if you do rename this, probably also need to rename attr_accessor on top and the readme

Reply:
got it from here: https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/database_configurations.rb#L38

Reply:
in this case it's one database specification (i.e. 'primary' or 'read_replica' or 'engine1') if using multiple databases